config/jobs/kubernetes-sigs/service-apis/service-apis-config.yaml

```diff
@@ -0,0 +1,30 @@
+presubmits:
+  kubernetes-sigs/service-apis:
+  - name: pull-service-apis-verify
+    annotations:
+      testgrid-dashboards: sig-network-service-apis
+      testgrid-tab-name: verify
+    decorate: true
+    path_alias: sigs.k8s.io/service-apis
+    always_run: true
```

Comment:
note that these will be immediately required for merge so we'll want to make sure they work.

without knowing the details of the makefile these look fine, but we'll find out :-)

Reply:
good catch